stories/04-features/Authentication.stories.mdx

```diff
@@ -0,0 +1,18 @@
+<Meta title="Features|Authentication" />
+
+# Authentication
+
+## Basic Authentication
+Basic auth can be enabled by enabling two environment variables. They will automatically apply the Authorization header for requests to the app. This will prompt the user to enter the valid credentials to access the site.
```

Comment:
This actually requires one more env var now, `BASIC_AUTH=true`. This is because basic auth is also used to protect cache endpoints, so there has to be a way to do the two separately (protect endpoints vs. put the site behind basic auth).